packages/fx-core/src/component/middleware/questionMW.ts

```diff
@@ -7,9 +7,12 @@ import { TOOLS } from "../../core/globalVars";
 import { QuestionNodes, questionNodes } from "../../question";
 import { traverse } from "../../ui/visitor";
 
-export function QuestionMW(key: keyof QuestionNodes): Middleware {
+export function QuestionMW(key: keyof QuestionNodes, fromAction = false): Middleware {
   return async (ctx: HookContext, next: NextFunction) => {
     const inputs = ctx.arguments[0] as Inputs;
```

Comment:
if fromAction is true, the inputs here is DriverContext, is there any issue if we use it as Inputs type?

Reply:
Seems no error when building and running. Do you think we need to mock an input and set platform, xxx into it?